python/ray/data/dataset.py

```diff
@@ -2211,11 +2211,14 @@ def to_tf(
         Time complexity: O(1)
 
         Args:
-            label_column (str): The name of the column used as the label
-                (second element of the output tuple).
-            output_signature (Tuple[tf.TypeSpec, tf.TypeSpec]): A 2-element
-                tuple of `tf.TypeSpec` objects corresponding to
-                (features, label).
+            output_signature (Tuple[tf.TypeSpec, tf.TypeSpec]): If
+                ``label_column`` is specified, a 2-element
+                tuple of ``tf.TypeSpec`` objects corresponding to
+                (features, label). Otherwise, a single ``tf.TypeSpec``
+                corresponding to features tensor.
+            label_column (Optional[str]): The name of the column used as the label
+                (second element of the output tuple). If not specified, output
```

Comment:
ditto: output_signature is not necessarily a tuple after your change?

Reply:
It's explained in the second sentence that if left unspecified, output will be just one tensor. I clarified it further, lmk if it's good now